modules/serverless-config-replicas.adoc

```diff
@@ -0,0 +1,49 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-HA.adoc
+
+[id="serverless-config-replicas_{context}"]
+= Configuring high availability replicas on {ServerlessProductName}
+
+HA functionality is available by default on {ServerlessProductName} for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components. These components are configured with two replicas by default.
+
+You modify the number of replicas that are created per controller by changing the configuration of `KnativeServing.spec.highAvailability` in the KnativeServing custom resource definition.
+// This field also specifies the minimum number of _activators_ if you are using the Horizontal Pod Autoscaler (HPA). For more information about HPA, see
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
+* Installed the {ServerlessOperatorName} and Knative Serving.
+
+.Procedure
+
+. In the {product-title} web console *Administrator* perspective, navigate to *OperatorHub* -> *Installed Operators*.
++
+image::serving-installed-operator.png[Installed Operators page]
++
+. Check that you have selected the `knative-serving` namespace.
++
+. Click *Knative Serving* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Serving* tab.
++
+image::serving-tab-created.png[Knative Serving tab]
++
+. Click on *knative-serving*, then go to the *YAML* tab in the *knative-serving* page.
++
+image::serving-YAML-HA.png[Knative Serving YAML]
++
+. Edit the custom resource definition YAML:
++
+.Example YAML
+[source,yaml]
+----
+spec:
+  high-availability:
+    replicas: 1
+----
++
```

Comment:
If the topic is _Configuring high availability replicas_, should the YAML example show something other than `1` which disables HA?

Reply:
Changed it to 3 :)